### ISSUE TITLE
Standalone detection

### ISSUE THREAD

**Issue by pjsample** [AlphonsG/EPIC-BBox-Cell-Tracking#12]
Hello, I'm trying to run the detection program alone as follows:
```
epic detection --save-dets --vis-dets --num-frames 2  --num-workers 8  --always <image_dir>  <EPIC_dir>/demo_config.yaml
```

It logs the message `[epic.cli:24 INFO] Starting.` but then exits without doing anything. Any thoughts on why?

**Comment by AlphonsG** [comment]
Hello, thanks for reporting this bug. I've made a patch for it. Please switch to the new `bug/detection` branch, then pull the changes and attempt to run again. If it works I'll merge with main.

**Comment by pjsample** [comment]
I just ran a small test and it works! Thanks again for the help.

You might be interested to know that I was able to get your code up and running easily using an AWS Deep Learning AMI. I used one of their conda environments configured with Python 3.8 and pytorch and then simply followed your installation instructions from step 5 onward. I then edited the config file to use the GPU of my EC2 instance. Much easier than dealing with CUDA installations on your own.
